packages/@sanity/default-layout/package.json

```diff
@@ -32,7 +32,7 @@
   "dependencies": {
     "@portabletext/react": "^1.0.0",
     "@reach/auto-id": "^0.13.2",
-    "@sanity/asset-utils": "^1.2.0",
+    "@sanity/asset-utils": "^1.2.5",
```

Comment:
Do we need to add this dependency for the default-layout? 🤔

Reply:
It's used in `PortableTextContent` already - this just changes the version

Reply:
Of course, for some reason when I first saw it I didn't see the red on the previous version!